### PR TITLE
Adds examples of how Artsy values are manifested

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR adds an example to each of the Artsy values that helps contextualize how the values affect our day-to-day work. This work fell out of our meeting on engineering team culture last week.

We could add more examples, but I like the clarity of only having one per value. These examples are up for discussion – I just wrote the first thing that came to mind – so if anyone has feedback please sound off in the comments 👇

### CODE REVIEW

**Review by orta — APPROVED** [review]

File: meta/what_is_artsy.md [hunk 1]
```diff
@@ -21,22 +21,32 @@ Artsy has 5 core values:
...
 
 We believe that art is fundamentally a positive force that can allow people to feel more inspired, connected, and present. Sadly, today many people’s perception of the art world is associated with negative feelings, such as exclusion, confusion, or judgment. We actively work to counteract these negative feelings by making our users’ interactions with art as inclusive and positive as possible.
 
+> **Example**: We begin our weekly engineering standups with "props"; everyone is encouraged to recognize and celebrate contributions that team members have made since the last standup.
+
 ### Openness
 
 We believe that openness allows for fuller self-expression and for art to be more fully experienced. We believe that being truly open in our communication—both giving and receiving—leads to more informed decisions and more productive relationships, which leads to better business.
 
+> **Example**: We encourage our engineering team members to be open about asking for help; our weekly standups have a 'requests for pairing' where we encourage folks to ask for help on anything they've been struggling with.
```
Comment (craigspaeth) [inline meta/what_is_artsy.md:48]: I think our Open Source by Default mantra might be a stronger example for this one.

**Review by craigspaeth — APPROVED** [review]

**Comment by craigspaeth** [comment]
👏  this is great. Thanks for adding. I only have one minor suggestion and you can feel free to merge when ready.

File: meta/what_is_artsy.md (hunk 1 above)
Comment (ashfurrow) [reply meta/what_is_artsy.md:48]: Yeah yeah, I kind of wanted to go that route but we also spend a lot of the rest of this document discussing Open Source by Default. I think using the non-OSS meaning of Openness helps clarify that it's not just about open source. What do you think?

Comment (craigspaeth) [reply meta/what_is_artsy.md:48]: Oh yeah, good point. Forgot this doc goes all into that later. 👍  to keeping as-is then.

File: meta/what_is_artsy.md [hunk 2]
```diff
@@ -21,22 +21,32 @@ Artsy has 5 core values:
...
 
+> **Example**: We work as engineers, but we aren't defined by that role; Artsy holds an annual [Salon](http://www.dictionary.com/browse/salon) and we encourage our engineers (and indeed, all our colleagues) to submit work. Engineer contributions to this event have included generative pixel art, interactive video game, photo, video, sculpture, painting, live music and cartographic pieces.
+
 ### People are Paramount
 
 We believe in empowering each of our team members to feel like the CEO of their area of responsibility. People are our company’s most valuable asset, and this is reflected in the high degree of responsibility and freedom we afford every team member. This approach guides how Artsy treats our people in all other realms as well.
 
+> **Example**: We hold a weekly team lunch with a healthy, delicious meal; after introducing any new colleagues, our People Ops team encourages everyone to sit and eat with people they don't normally get to talk to.
```
Comment (acjay) [inline meta/what_is_artsy.md:30]: I wonder if S'up would be a good example here, especially since it came out of engineering.

**Review by acjay — COMMENTED** [review]
These are great, and I think we should ship.

But I do wonder if it would be even more powerful to show places where the values guided decisions where there are real or perceived tradeoffs (like how asking for help means accepting some ego risk; or the tension between OSS and proprietary IP)?

**Comment by ashfurrow** [comment]
Agreed @acjay – I've opened #7 to follow up.

**Pull request merged**